activestorage/lib/active_storage/validations/content_type_validator.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module ActiveStorage
+  module Validations
+    class ContentTypeValidator < ActiveModel::EachValidator
+      def validate_each(record, attribute, value)
+        files = record.send(attribute)
+
+        return true unless files.attached?
+        return true if types.empty?
+
+        files = Array.wrap(files)
+
+        errors_options = { authorized_types: types_to_human_format }
+        errors_options[:message] = options[:message] if options[:message].present?
+
+        files.each do |file|
+          unless content_type_valid?(file)
+            errors_options[:content_type] = content_type(file)
+
+            record.errors.add(attribute, :content_type_invalid, errors_options)
+            return
```

Comment:
Is this supposed to return here? Is this so that the error is only added once per input?

Reply:
Hey sorry! code is updated in Gem. I will update here soon or open new PR.
https://github.com/igorkasyanchuk/active_storage_validations/tree/master/lib/active_storage_validations